index.html

```diff
@@ -3293,22 +3298,22 @@ <h3>Status</h3>
 
         <p>
 If the <code>credentialStatus</code> property is available, the status of the
-claim MUST be in good standing according to the <code>credentialStatus</code>
-<a>type</a> definition for the <a>credential</a>. For example, a <a>verifier</a>
-can ensure that the status has not been withdrawn for cause by the
-<a>issuer</a>.
+a <a>verifiable credential</a> MUST be in good standing according to the
+<code>credentialStatus</code> <a>type</a> definition for the
+<a>verifiable credential</a>. For example, a <a>verifier</a> can ensure that the
```

Comment:
Whether a 'revoked' or 'expired' credential is still valid is primarily determined by the verifier's use case, which may not follow the 'good standing' according to the credentialStatus. Having MUST here is inappropriate.
I recognize this isn't something you've introduced, but it is an error in the verification section that needs fixing.

Reply:
Changed the MUST to a MAY and that the VC be evaluated according to the credentialStatus type definition AND the verifier's own status evauation criteria.